app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,8 @@
   <body class="h-screen">
     <%=render "layouts/menu"%>
     <main class="container mx-auto mt-5 flex">
+      <p class="notice"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
```

Comment:
Where are these classes and methods defined?

Reply:
I don't have them at the views. I need to  add them. Check the book for fine tuning its behavior and finding out the best way with Tailwind.